src/System.Private.Xml.Linq/src/System.Private.Xml.Linq.csproj

```diff
@@ -9,15 +9,15 @@
     <DefineConstants>$(DefineConstants);SILVERLIGHT</DefineConstants>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'net46'">None</ResourcesSourceOutputDirectory>
```

Comment:
Orthogonal to your change but do you know if there is any reason this library cannot be one-to-one with the System.Xml.XDocument contract? I didn't believe it had any circular dependencies as it was already implemented independently on the full .NET framework.